doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
+
+  (define args
+    (command-line
+     #:program (short-program+command-name)
+     #:once-any
+     [("-b" "--binding") b
+      "Check the documentation for a specific binding"
+      (binding (string->symbol b))]
+     [("-r" "--ratio") r
+      "Specify required documentation ratio"
+      (ratio (string->number r))]
+     [("-s" "--skip") s
+      "Specify regex of bindings to ignore"
+      (ignore (regexp s))]
+     #:args (file . files)
+     (cons file files)))
+
+  (for ([a (in-list args)])
+    (let/ec break
+
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (set! a (string->symbol a)))])
+        (namespace-require a))
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (fprintf (current-error-port) "Module ~a can not be loaded~n" a)
+                                   (error-on-exit? #t)
+                                   (break))])
+        (namespace-require a))
+
+      (cond [(binding)
```

Comment:
Sorry, I'm a bit confused, are you just saying this should be a helper function passing in `(binding)`? If so okay.

Although I feel like I should point out that I doubt calling `(binding)` 6 times here would slow things down. I don't really know the performance characteristics of parameters (other than I suspect they are slow), but calling (binding) isn't really on the critical path.

On the other hand if you're just concerned about readability, I guess that's another issue which than goes back to helper functions, which again, I'd be happy (albeit a bit begrudgingly ;) ) to add for you if you'd like. Just let me know.